includes/class-wc-payments.php

```diff
@@ -1134,6 +1138,7 @@ public static function ajax_init_platform_checkout() {
 				'return_url'                     => wc_get_cart_url(),
 			],
 			'user_session'         => isset( $_REQUEST['user_session'] ) ? sanitize_text_field( wp_unslash( $_REQUEST['user_session'] ) ) : null,
+			'blocks_data'          => $blocks_data_extractor->get_data(),
```

Comment:
Would this fit better into `store_data`? Sorry I didn't ask in the previous review. Things work fine like this. Wondering because we include plugin information from the merchant store there, like the subscriptions one. But again, all good.

Reply:
> Would this fit better into store_data?

I agree, thanks for pointing it out. Changed it in https://github.com/Automattic/woocommerce-payments/pull/5483/commits/26c9a2b77b5be3fe634b8644b4ee684536abd95e

Reply:
Thanks!